app/controllers/stafftools/assignment_invitations_controller.rb

```diff
@@ -0,0 +1,23 @@
+module Stafftools
+  class AssignmentInvitationsController < StafftoolsController
```

Comment:
You can write this as `class Stafftools::AssignmentInvitationsController < StafftoolsController`.